### ISSUE TITLE
VB -> C#: Missing conversion for array index assignment

### ISSUE THREAD

**Issue by mrmonday** [icsharpcode/CodeConverter#365]
### Input code
```vbnet
Module Module1
    Sub Main()
        Dim x As New ArrayList
        x.Add("a")

        Dim xs(1) As String

        xs(0) = x(0)

    End Sub
End Module
```

### Erroneous output
```c#
namespace ConsoleApp10
{
    static class Module1
    {
        public static void Main()
        {
            ArrayList x = new ArrayList();
            x.Add("a");

            string[] xs = new string[2];

            xs[0] = x[0];
        }
    }
}
```

### Expected output
```c#
namespace ConsoleApp10
{
    static class Module1
    {
        public static void Main()
        {
            ArrayList x = new ArrayList();
            x.Add("a");

            string[] xs = new string[2];

            xs[0] = Conversions.ToString(x[0]);
        }
    }
}
```

### Details

Version in use: master @ 2483369355807050379e2385a4f22d16eb3437f7

**Comment by BrianFreemanAtlanta** [comment]
Attempting to look into this one.  When I get into `AnalyzeConversion` the typeInfo/vbType = "ErrorType ?"  Not sure if additional information is needed on the conversion or add a case for the ErrorType or take another tack.
If you'd like me to do a pull request with the test case in it I can do that.

**Comment by GrahamTheCoder** [comment]
If you see ErrorType or ErrorSymbol it generally means the compilation had errors.
If you look at `_semanticModel.GetDiagnostics()` you'll be able to see them.
In this case, adding an import should cover it:
```vbnet
Imports System.Collections

Public Module Module1
	Sub Main()
        Dim x As New ArrayList
        x.Add("a")

        Dim xs(1) As String

        xs(0) = x(0)

    End Sub
End Module
```

Other stuff:
* There's also a `_showCompilationErrors` flag I use for showing errors in the test output, but I recently broke it. It'll be fixed again as part of #60 
* Ideally that System.Collections import would have been automatically added for all conversions since it's a common one (and unused imports get removed at the end). Things like that should go into the list here: https://github.com/icsharpcode/CodeConverter/blob/master/ICSharpCode.CodeConverter/Util/VisualBasicCompiler.cs#L58 which if it "just works" would probably be a better fix rather than changing the test

**Comment by GrahamTheCoder** [comment]
Generally it's also worth just having a solution with a VB and a CS project open alongside the code converter for checking syntax.
